ChatRPG/Pages/CampaignPage.razor.cs

```diff
@@ -27,25 +27,33 @@ public partial class CampaignPage
 
     [Inject] private IConfiguration? Configuration { get; set; }
     [Inject] private IOpenAiLlmClient? OpenAiLlmClient { get; set; }
-    [Inject] private IPersisterService? PersisterService { get; set; }
     [Inject] private IJSRuntime? JsRuntime { get; set; }
     [Inject] private AuthenticationStateProvider? AuthenticationStateProvider { get; set; }
+    [Inject] private IPersisterService? PersisterService { get; set; }
+    [Inject] private ICampaignMediatorService? CampaignMediatorService { get; set; }
     [Inject] private GameInputHandler GameInputHandler { get; set; } = null!;
```

Comment:
My bad, lad os gøre den her consistent med de andre:
```suggestion
    [Inject] private GameInputHandler? GameInputHandler { get; set; }
```

Reply:
Det er fixed i nyeste commit 👍